xclim/stats.py

```diff
@@ -15,7 +18,9 @@ def fit(arr, dist='norm'):
     # Get the distribution object
     dc = getattr(stats, dist, None)
     if dc is None:
-        raise ValueError("Statistical distribution {} is not in scipy.stats.".format(dist))
+        e = "Statistical distribution {} is not in scipy.stats.".format(dist)
+        logging.exception(ValueError(e))
```

Comment:
Is that still necessary ?
```suggestion
```

Reply:
Sounds good. Let's try to get all these PR checked-in before the end of the week.